test/python/circuit/test_initializer.py

```diff
@@ -357,6 +357,20 @@ def test_from_labels(self):
             fidelity, self._desired_fidelity,
             "Initializer has low fidelity {:.2g}.".format(fidelity))
 
+    def test_from_int(self):
+        """Initialize from int."""
+        desired_sv = Statevector.from_label('110101')
+
+        qc = QuantumCircuit(6)
+        qc.initialize(53, range(6))
+        job = execute(qc, BasicAer.get_backend('statevector_simulator'))
+        result = job.result()
+        statevector = result.get_statevector()
+        fidelity = state_fidelity(statevector, desired_sv)
+        self.assertGreater(
+            fidelity, self._desired_fidelity,
+            "Initializer has low fidelity {:.2g}.".format(fidelity))
```

Comment:
Can you use the `Statevector` to do this check? 🙂 
```suggestion
        actual_sv = Statevector.from_instruction(qc)
        self.assertTrue(desired_sv == actual_sv)
```

Reply:
I also changed the `test_from_label ` above this one to use `Statevector`.

Done in https://github.com/Qiskit/qiskit-terra/pull/5418/commits/8a7f33f292af59659e661aff949c9e31a25ec79a